src/Essentials/src/Browser/Browser.ios.cs

```diff
@@ -16,7 +16,6 @@ public async Task<bool> OpenAsync(Uri uri, BrowserLaunchOptions options)
 			switch (options.LaunchMode)
 			{
 				case BrowserLaunchMode.SystemPreferred:
-					System.Diagnostics.Debug.Assert(!OperatingSystem.IsIOSVersionAtLeast(11));
 					await LaunchSafariViewController(uri, options);
```

Comment:
What happens if you run this code on iOS 11? Will it crash? If so, then we need to update the switch to maybe be an if and then make it also check version. If SystemPreferred && OSVersion(11) then use safari, else use external

Reply:
> What happens if you run this code on iOS 11? Will it crash? 

I do not have a device setup to test `iOS`, anyway all these warnings annotation/attribute based 

After removing the Assert there is no warning here anymore, I think it is because we increased the browser version to support ios11.0+, I can check that when I got access to my desktop

Reply:
The method `LaunchSafariViewController(Uri uri, BrowserLaunchOptions options)` below annotated with `UnsupportedOSPlatform("ios11.0")` because it is unconditionally calling `SFSafariViewController` constructor unsupported from iOS 11.0, I see there is a constructor supported from 11.0+ 
![image](https://user-images.githubusercontent.com/45579687/166309765-61975ab0-959d-4c8a-a0b9-3e0f9058eb8f.png)
So the correct handling is probably conditionally call those constructors, but I do not know what to pass for `SFSafariViewControllerConfiguration` parameter